flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/api/TableEnvironmentTest.scala

```diff
@@ -656,13 +656,48 @@ class TableEnvironmentTest {
   def testExecuteSqlWithShowDatabases(): Unit = {
     val tableResult1 = tableEnv.executeSql("CREATE DATABASE db1 COMMENT 'db1_comment'")
     assertEquals(ResultKind.SUCCESS, tableResult1.getResultKind)
-    val tableResult2 = tableEnv.executeSql("SHOW DATABASES")
+    val tableResult2 = tableEnv.executeSql("CREATE DATABASE db2 COMMENT 'db2_comment'")
+    assertEquals(ResultKind.SUCCESS, tableResult2.getResultKind)
```

Comment:
It's recommended  to use `assertThat` from assertj pacakge. If you think so, please  keep consistent in the rest of the related changed lines.

Reply:
The method `assertThat` is marked as `@Deprecated` in `junit-4.13.2.jar`.
```java
  @Deprecated
  public static <T> void assertThat(T actual, Matcher<? super T> matcher) {
      assertThat("", actual, matcher);
  }
```

Reply:
org.assertj.core.api.Assertions

Reply:
`org.assertj.core.api.Assertions.assertThat` is not suitable for comparing enumeration variables. And I think we'd better keep consistent style in the same file.

Reply:
Why is it not suitable for enums?

Reply:
@ZhangChaoming ping

Reply:
@slinkydeveloper I can not find a suitable method to comparing enumeration variables by using `org.assertj.core.api.Assertions.assertThat`.

Reply:
have a try on `assertThat(enumAObj).isEqualTo(enumBObj);`

Reply:
@RocMarshal  I tried, but it did not work.

Reply:
@RocMarshal Can you give me some suggestions?

Reply:
Just have a try on replacing the code line 660 with `assertThat(tableResult2.getResultKind).isEqualTo(ResultKind.SUCCESS)` and discard the warnings from IDE, then run the test case.

Reply:
@RocMarshal Thanks, it really works !